docs/jwst/emicorr/description.rst

```diff
@@ -0,0 +1,58 @@
+Description
+===========
+
+:Class: `jwst.emicorr.EmiCorrStep`
+:Alias: emicorr
+
+Overview
+--------
+The ``emicorr`` step corrects for known noise patterns in the raw MIRI data.
+The majority of the MIRI subarrays have an 390 Hz or other electromagnetic
+interference (EMI) noise pattern in the raw data. The known frequencies to
+correct for are in the EMI reference file, under the key ``frequencies``.
+The effect of these EMI frequencies is to imprint each into the rate
+images. For short integrations in ``LRSSLITLESS`` the correlated noise from
+this is quite apparent in the rate images. For longer integrations the net
+effect is to increase the read noise by about 20\%.
+
+The process to do the correction is the following (repeated
+recursively for each discrete EMI frequency desired):
+1. Read image data.
+2. Make very crude slope image and fixed pattern "super" bias for each
+integration, ignoring everything (nonlin, saturation, badpix, etc).
+3. Subtract scaled slope image and bias from each frame of each integration.
+4. Calculate phase of every pixel in the image at the desired EMI frequency
+(e.g. 390 Hz) relative to the first pixel in the image.
+5. Make a binned, phased amplitude (pa) wave from the cleaned data (plot
+cleaned vs phase, then bin by phase).
+6.* Measure the phase shift between the binned pa wave and the input
+reference wave
+7.* Use look-up table to get the aligned reference wave value for each pixel
+(the noise array corresponding to the input image).
+*) Alternately, use the binned pa wave instead of the reference wave to
+"self-correct"
```

Comment:
```suggestion
6. Measure the phase shift between the binned pa wave and the input
reference wave
7. Use look-up table to get the aligned reference wave value for each pixel
(the noise array corresponding to the input image). Alternately, use the binned pa wave instead of the reference wave to "self-correct".
```
I'm not sure if the `*` here are causing the docs failures (`WARNING: Inline emphasis start-string without end-string`).
I made what I thought might be a correct suggestion fo `7` above but please correct if if I changed the meaning.